nexus/src/app/background/tasks/fm_sitrep_gc.rs

```diff
@@ -264,9 +266,12 @@ mod tests {
                 comment: format!("test sitrep v{i}; orphan {i}"),
                 time_created: Utc::now(),
                 parent_sitrep_id,
+                // TODO(eliza): we should populate cases and assert they get
+                // cleaned up...
```

Comment:
Since we're adding deletion operations in this PR -- WDYT about addressing this comment before we merge?